JavaScript-Files/test_ui.js

```diff
@@ -0,0 +1,32 @@
+function Player(name) {
+	this.name    = name,
+	this.money   = 15,
+	this.workers = 0;
+}
+
+var you  = new Player("You");
+var you2 = new Player("Rival");
+var you3 = new Player("Arch nemesis");
+var ai1  = new Player("Ai1");
+var ai2  = new Player("Ai2");
+var ai3  = new Player("Ai3");
+
+var begin_round = function() {
+	// Dice
+	var dice = Math.floor(Math.random() * 6) + 1;
+	
+	// Buy workers for player You
+	var buy_workers_you = function() {
+		var max = you.money/20;
+		var howMany = prompt("How many workerws would you like to buy? The most you can buy is " + max);
+		var ask = function() {
+			if(howMany > max) {
+				alert("Sorry! You cannot buy that many workers.");
+				ask();
+			}else{
+				you.workers += howMany;
+				you.money - howMany * 20;
```

Comment:
__[jshint]__
* Expected an assignment or function call and instead saw an expression.

<!-- Comment by [SideCI](https://sideci.com) -->